tests/v1/test_auth.py

```diff
@@ -8,14 +8,14 @@
 import pytest
 import pytest_asyncio
 
-from aiohubspace.v1 import auth
+from aioafero.v1 import auth
 
 current_path = pathlib.Path(__file__).parent.resolve()
 
 
 @pytest.fixture(scope="function")
 def hs_auth():

```

Comment:
Maybe rename to afero_auth

Reply:
I went back and forth on this one. Its validating hubspace things so `hs_auth` validates hubspace things. I think I will leave this as all tests currently pass.